docs/guides/q-ctrl-performance-management.ipynb

```diff
@@ -29,88 +29,27 @@
-    "\n",
-    "<Admonition type=\"caution\">Fire Opal Performance Management accepts abstract circuits, in contrast to the native Qiskit Runtime primitives, which only accept circuits that are written in the target backend’s Instruction Set Architecture (ISA). For best results, do not transpile circuits before submitting via the Performance Management function. </Admonition>\n",
-    "\n",
-    "\n",
-    "### Sampler inputs\n",
-    "| Name       | Type                                          | Description                                                                                                 | Required | Example                                                                                                             |\n",
-    "|------------|-----------------------------------------------|-------------------------------------------------------------------------------------------------------------|----------|---------------------------------------------------------------------------------------------------------------------|\n",
-    "| pubs       | `SamplerPubLike` or `list[SamplerPubLike]`   | One or more tuples containing the inputs listed under `SamplerPubLike` components.                          | Yes      | `(circuit, parameter_values)`                                                                                |\n",
-    "| instance   | str                                           | The hub/group/project to use in that format.                                                               | Yes      | `\"hub1/group1/project1\"`                                                                                           |\n",
-    "| run_options| dict                                          | Input options which includes the following: (Optional) `backend_name`: str, `shots`: int. Defaults to least busy backend and 4000 shots. | No       | `{\"backend_name\": backend_name, \"shots\": 2048}`                                                                   |\n",
-    "\n",
-    "**`SamplerPubLike` components:**\n",
-    "- A single circuit in the OpenQASM 2.0 or 3.0 string format.\n",
-    "- (Optional) A collection of parameter values to bind the circuit against.\n",
-    "- (Optional) A dictionary of run options, such as the shot count.\n",
+    "Fire Opal's pipeline eliminates the need for additional overhead, such as increased quantum runtime or extra physical qubits. Note that classical processing time remains a factor (refer to the Benchmarks section for estimates, where \"Total time\" reflects both classical and quantum processing). In contrast to error mitigation, which requires overhead in the form of sampling, Fire Opal's error suppression works at both the gate and pulse levels to address various sources of noise and to prevent the likelihood of an error occurring. By preventing errors, the need for expensive post-processing is eliminated.\n",
```

Comment:
```suggestion
    "Fire Opal's pipeline eliminates the need for additional overhead, such as increased quantum runtime or extra physical qubits. Note that classical processing time remains a factor (refer to the [Benchmarks](#benchmarks) section for estimates, where \"Total time\" reflects both classical and quantum processing). In contrast to error mitigation, which requires overhead in the form of sampling, Fire Opal's error suppression works at both the gate and pulse levels to address various sources of noise and to prevent the likelihood of an error occurring. By preventing errors, the need for expensive post-processing is eliminated.\n",
```